buildSrc/src/main/groovy/io.deephaven.java-conventions.gradle

```diff
@@ -74,18 +76,85 @@ tasks.withType(JavaCompile).configureEach {
   }
 }
 
+def tier4CompilationExcludes = [
+  'io.deephaven.engine.table.impl.by.ssmpercentile.DoublePercentileTypeMedianHelper':['setResult'],
+  'io.deephaven.engine.table.impl.by.ssmpercentile.FloatPercentileTypeMedianHelper':['setResult'],
+  'io.deephaven.engine.table.impl.by.ssmpercentile.SsmChunkedPercentileOperator':['addChunk'],
+  'io.deephaven.engine.table.impl.by.IncrementalChunkedOperatorAggregationStateManager':['buildTable'],
+  'io.deephaven.engine.table.impl.RightIncrementalChunkedCrossJoinStateManager':['addToIndex', 'decorationProbe'],
+  'io.deephaven.engine.table.impl.IncrementalChunkedNaturalJoinStateManager':['buildTable', 'decorationProbe'],
+]
+def createCompilerDirectives = tasks.register('createCompilerDirectives') {
+  def c2ExcludesFile = project.layout.buildDirectory.file('c2-excludes.txt')
```

Comment:
Agree w/ your comment elsewhere, I think we should name this something else - maybe `dh-compiler-directives.txt`?

Reply:
Perfect, I'll use that - was mostly looking for someone else to make a suggestion...